ppb/features/default_sprites.py

```diff
@@ -21,8 +25,13 @@ class TargetSprite(ppb.Sprite):
     def on_update(self, update_event, signal):
         if self.max_speed < self.min_speed:
             raise ValueError("TargetSprite maximum speed cannot be less than minimum speed.")
+        if self.exponential_speed > 1.0:
+            raise ValueError("TargetSprite exponential speed cannot be greater than 1.")
```

Comment:
```suggestion
            raise ValueError(f"{type(self).__name__} exponential speed cannot be greater than 1.")
```

This will let subclassed versions retain their class name in the error message.

Reply:
Did the same for the max/min speed error.